src/sponge/mod.rs

```diff
@@ -4,7 +4,10 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-mod sponge;
+pub mod sponge;
+pub mod truncated;
 
 pub use sponge::sponge_gadget as gadget;
 pub use sponge::sponge_hash as hash;
+pub use truncated::gadget as truncated_gadget;
+pub use truncated::hash as truncated_hash;
```

Comment:
I don't think those are needed anymore; the lines 10 and 11 exists to normalize but we don't need to normalize the `truncated` module (is already).

To be clear, the API that I'd like to have is the following one:
```rust
use dusk_poseidon::sponge::truncated;
…
truncated::hash(…)
```

In the same way we have now (thanks to the normalization):
```rust
use dusk_poseidon::sponge;
…
sponge::hash(…)
```

And if we need both:
```rust
use dusk_poseidon::sponge::{self, truncated};
```